pom.xml

```diff
@@ -72,8 +72,8 @@
                         <configuration>
                             <rules>
                                 <requireJavaVersion>
-                                    <version>[1.8.0,1.9.0)</version>
-                                    <message>You need JDK8 or lower</message>
+                                    <version>1.8.0</version>
+                                    <message>You need JDK8 or higher</message>
```

Comment:
I think this change would limit it to 1.8.0 and not allow 1.8.1,
not that I ever expect there to be a 1.8.1.  As currently written,
it allows 1.8.*.  "or lower" is probably wrong.

Reply:
https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html:
> 1.0 | x >= 1.0 * The default Maven meaning for 1.0 is  everything (,) but with 1.0 recommended. Obviously this doesn't work for  enforcing versions here, so it has been redefined as a minimum version.

Reply:
Sorry, haven't had time to get back to this to verify that the fix works as expected.

Reply:
There are still two commits, 1<sup>st</sup> one assuming wrong implementation in pom, and 2<sup>nd</sup> assuming wrong requirement with correctly expressed rule. I don't know which one is right, so feel free to merge whichever you prefer or none.